### DISCUSSION TITLE
Some Traitor missions AND main missions can BOTH be completed

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14421] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Some traitor missions that try to sabotage the main missions can be completed along with the main mission, while others cannot (throwing out cargo)

* **bakingstation** (blow up beacon reactor), you can blow up the reactor AND repair the station after, completing both missions. 

Turn reactor noninteractable
```xml
      <CheckConditionalAction targettag="beaconreactor" targetitemcomponent="Reactor" MeltedDownThisRound="true">
        <Failure>
          <GoTo name="waitforexplosion" />
        </Failure>
        <Success>
          <StatusEffectAction targettag="beaconreactor">
            <StatusEffect target="This" NonInteractable="true" setvalue="true" />
          </StatusEffectAction>
        </Success>
      </CheckConditionalAction>
```

Also the text says:
_`The beacon station you're supposed to visit might have a nuclear reactor that could be overloaded, causing a huge explosion and radiation to harm those Jove-forsaken crawlers and mudraptors for a long time to come.`_

But reactors don't spawn the item `nuclearaftereffectemitter` when destroyed....

* **vippoison** and **vipbombing**  you can poison and bomb the vip but also keep them alive, completing both missions

### Version

v1.5.9.1 (Summer Update Hotfix 2)

**Comment by Regalis11** [comment]
I don't see an issue with this.
